CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ### Fixes
 
+* `[enzyme]` Allow matching of Errors against plain objects
```

Comment:
not enzyme, wrong matcher library :D

Reply:
https://github.com/facebook/jest/pull/5614

Reply:
ah, nice!